_posts/2023-01-11-install-vpn-from-file.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Installing an OpenVPN File Without Using Terminal
+description: This guide provides a step-by-step process for installing an OpenVPN file without using the terminal. It is designed to help users who may be uncomfortable with command-line interfaces or who simply prefer a more user-friendly approach.
+date: 2023-01-11
+layout: article
+authors: 
+    - [Jacob Gonzales](https://kutt.it/jacobgonzales)
```

Comment:
Nice work, I will review this soon. The authors field only supports GitHub usernames currently.

Reply:
> Nice work, I will review this soon. The authors field only supports GitHub usernames currently.

oh okay, my github username will do! I plan to add more guides!

Reply:
> I plan to add more guides catered toward non-tech-savvy people

Feel free to let us know about the guides you are writing in the #docs-channel in our Discord server (Link in README) so that we can track these articles [here](https://github.com/orgs/Vanilla-OS/projects/2).

Reply:
> > I plan to add more guides catered toward non-tech-savvy people
> 
> Feel free to let us know about the guides you are writing in the #docs-channel in our Discord server (Link in README) so that we can track these articles [here](https://github.com/orgs/Vanilla-OS/projects/2).

I would definitely, if it wasn't discord. Would you like me to make a issue before making new guides so you can add them to the documentation progress?

Reply:
> I would definitely, if it wasn't discord... Would you like me to make a issue before making new guides so you can add them to the documentation progress?

Yeah, that would do, @mirkobrombin can you add this person to the docs team?

For this, I have created [this](https://github.com/Vanilla-OS/handbook/issues/40), you can add "Closes #40" to this PRs Description. In future, once you are part of the team you can directly add entries to the project and create issues.

Reply:
> 

great! by adding directly do you mean, I'll be able to push post directly to the main directory of this repo?

Reply:
> > 
> 
> great! by adding directly do you mean, I'll be able to push post directly to the main directory of this repo?

No, You can access the project with todo list and progresd. Being part of the docs team you can start  discussions internally since you don't prefer Discord.

Reply:
You can discuss here after joining

![image](https://user-images.githubusercontent.com/26346867/211982042-e1c5dae6-87f6-45f2-a92a-33f6ea0ea2e1.png)

Currently there are 0 discussion 😅